docs/samples/browser-plugin-meetings/app.js

```diff
@@ -59,10 +55,6 @@ tokenElm.addEventListener('change', (event) => {
   localStorage.setItem('date', new Date().getTime() + (12 * 60 * 60 * 1000));
 });
 
-if (isSafari || isiOS) {
-  document.getElementById('sendShareToggle').disabled = true;
```

Comment:
this is just something I've found - it prevents the sample app from working in Safari at all, because "sendShareToggle" doesn't exist anymore